semeio/jobs/scripts/csv_export2.py

```diff
@@ -29,5 +25,21 @@ def main(args):
     print("{} csv-export written to {}".format(args.time_index, args.outputfile))
 
 
+def csv_export_parser():
+    """Setup parser"""
+    parser = argparse.ArgumentParser()
```

Comment:
Will this argparse documentation (there is none atm) be plugged into ert as well?

Reply:
Yes it will, will show up something like: 
```
'usage: CSV_EXPORT2 [-h]
                            runpathfile outputfile time_index column_keys
                            [column_keys ...]

positional arguments:
  runpathfile
  outputfile
  time_index
  column_keys

optional arguments:
  -h, --help   show this help message and exit
```